rpc_jobs/multi_node_aio_leapfrog.yml

```diff
@@ -0,0 +1,120 @@
+- project:
+    name: 'Multi-Node-AIO-Leapfrog-Testing'
+    series:
+      - newton-r11.1.18:
+          UPGRADE_FROM_REF: "r11.1.18"
+          branch: "r14.2.0"
+          branches: "r14.2.0"
+          KIBANA_SELENIUM_BRANCH: "newton"
+          SERIES_USER_VARS: |
+            tempest_test_sets: 'scenario defcore api heat_api smoke'
```

Comment:
The `SERIES_USER_VARS` in newton [here](https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/multi_node_aio.yml#L27-L31) are significantly different.  Is that by design?

Reply:
The remaining vars were copied based on it being for kilo, but if it's newton we're building for, I probably should grab those vars for newton and update it for that as well.